UI/frontend-plugins/aja-output-ui/CMakeLists.txt

```diff
@@ -1,145 +1,81 @@
+          ${CMAKE_SOURCE_DIR}/plugins/aja/aja-enums.hpp
+          ${CMAKE_SOURCE_DIR}/UI/combobox-ignorewheel.cpp
+          ${CMAKE_SOURCE_DIR}/UI/combobox-ignorewheel.hpp
+          ${CMAKE_SOURCE_DIR}/UI/double-slider.cpp
+          ${CMAKE_SOURCE_DIR}/UI/double-slider.hpp
+          ${CMAKE_SOURCE_DIR}/UI/properties-view.hpp
+          ${CMAKE_SOURCE_DIR}/UI/properties-view.cpp
+          ${CMAKE_SOURCE_DIR}/UI/properties-view.moc.hpp
+          ${CMAKE_SOURCE_DIR}/UI/qt-wrappers.cpp
+          ${CMAKE_SOURCE_DIR}/UI/qt-wrappers.hpp
+          ${CMAKE_SOURCE_DIR}/UI/spinbox-ignorewheel.cpp
+          ${CMAKE_SOURCE_DIR}/UI/spinbox-ignorewheel.hpp
+          ${CMAKE_SOURCE_DIR}/UI/slider-ignorewheel.cpp
+          ${CMAKE_SOURCE_DIR}/UI/slider-ignorewheel.hpp
+          ${CMAKE_SOURCE_DIR}/UI/vertical-scroll-area.cpp
+          ${CMAKE_SOURCE_DIR}/UI/vertical-scroll-area.hpp)
+
+target_link_libraries(aja-output-ui PRIVATE OBS::libobs OBS::frontend-api
+                                            Qt::Widgets AJA::LibAJANTV2)
+
+if(TARGET AJA::LibAJANTV2_Debug)
+  target_link_libraries(aja-output-ui PRIVATE debug AJA::LibAJANTV2_Debug)
 endif()
 
-qt5_wrap_ui(aja-output-ui_UI_HEADERS
-	${aja-output-ui_UI})
-
-add_library(aja-output-ui MODULE
-	${aja-output-ui_HEADERS}
-	${aja-output-ui_SOURCES}
-	${aja-output-ui_UI_HEADERS})
-
-set(aja_LIBRARIES
-		$<IF:$<CONFIG:Debug>,${LIBAJANTV2_DEBUG_LIBRARIES},${LIBAJANTV2_LIBRARIES}>
-		libobs)
-
-# macOS
-if(APPLE)
-	set(aja_COMPILE_DEFS
-			AJAMac
-			AJA_MAC)
-
-	find_library(IOKIT_FRAMEWORK Iokit)
-	find_library(COREFOUNDATION_LIBRARY CoreFoundation)
-	find_library(APPKIT_FRAMEWORK AppKit)
-
-	list(APPEND aja_LIBRARIES
-			${IOKIT_FRAMEWORK}
-			${COREFOUNDATION_LIBRARY}
-			${APPKIT_FRAMEWORK})
-	# Windows
-elseif(WIN32)
-	set(aja_COMPILE_DEFS
-			AJA_WINDOWS
-			_WINDOWS
-			WIN32
-			MSWindows)
-
-	if(CMAKE_BUILD_TYPE STREQUAL Debug)
-		list(APPEND aja_COMPILE_DEFS
-				_DEBUG)
-	else()
-		list(APPEND aja_COMPILE_DEFS
-				NDEBUG)
-	endif()
-
-	list(APPEND aja_LIBRARIES
-			ws2_32.lib
-			setupapi.lib
-			Winmm.lib
-			netapi32.lib
-			Shlwapi.lib)
-	# Linux
-elseif(UNIX AND NOT APPLE)
-	set(aja_COMPILE_DEFS
-			AJA_LINUX
-			AJALinux)
+if(OS_MACOS)
+  find_library(IOKIT_FRAMEWORK Iokit)
+  find_library(COREFOUNDATION_LIBRARY CoreFoundation)
+  find_library(APPKIT_FRAMEWORK AppKit)
+
+  target_link_libraries(aja-output-ui PRIVATE ${IOKIT} ${COREFOUNDATION}
+                                              ${APPKIT})
+elseif(OS_WINDOWS)
+  set(MODULE_DESCRIPTION "OBS AJA Output UI")
+  configure_file(${CMAKE_SOURCE_DIR}/cmake/bundle/windows/obs-module.rc.in
+                 aja-output-ui.rc)
+  target_sources(aja-output-ui PRIVATE aja-output-ui.rc)
+
+  target_link_libraries(aja-output-ui PRIVATE ws2_32.lib setupapi.lib Winmm.lib
+                                              netapi32.lib Shlwapi.lib)
+else()
+  find_package(X11 REQUIRED)
+  target_link_libraries(aja-output-ui PRIVATE X11::X11)
```

Comment:
`Qt${QT_VERSION}::GuiPrivate` is missing.

Reply:
Doesn't seem necessary, wouldn't it break compilation already if it's required?

Reply:
Not built on Ubuntu

Reply:
Right.. 🤦🏻‍♂️

Will add that tomorrow, I can't stand looking at CMake code for another second for today..